### PR TITLE
projects/serializers: sending project.information html formatted

### PR DESCRIPTION
**Pull request by khamui**

### CODE REVIEW

**Review by goapunk — CHANGES_REQUESTED** [review]
the serializer tests now fail and need fixing.

File: tests/projects/test_app_project_api.py [hunk 1]
```diff
@@ -89,10 +89,11 @@ def test_app_project_api_single_poll_module(
 @pytest.mark.django_db
 def test_app_project_serializer(project_factory, module_factory, phase_factory,
                                 apiclient):
+    html_string = '<p>text with a <strong>bold</strong> bit</p>'
```
Comment (fuzzylogic2000) [inline tests/projects/test_app_project_api.py:92]: Even though, we do not remove the tags anymore, you could test that the strip works and removes whitespace.

**Review by fuzzylogic2000 — COMMENTED** [review]
Nice! Could you now also squash the commits? (When you just do a fixup with the second one, you will not have the weird commit message with "squash" in it.)

**Comment by khamui** [comment]
@fuzzylogic2000 oh makes sense. Now it tests the strip of whitespaces

**Pull request merged**